larray/io/excel.py

```diff
@@ -171,12 +173,16 @@ def _ipython_key_completions_(self):
             return list(self.sheet_names())
 
         def __getitem__(self, key):
+            if isinstance(key, Group):
+                key = re.sub('[\\/?*\[\]:]', '_', str(_to_tick(key)))
```

Comment:
for such a "fixed" regex, I prefer to compile the regexp and store it as a module variable

Reply:
OK.